talaiot/src/main/kotlin/com/cdsap/talaiot/configuration/FilterConfiguration.kt

```diff
@@ -17,6 +21,9 @@ import groovy.lang.Closure
  *      includes = arrayOf("feature.*")
  *      excludes = arrayOf("utils.*")
  *  }
+ *  threshold{
+ *      maxValue = 3000
+ *      minValue = 10
  * }
  */
```

Comment:
A missing bracket for the definition of a filter configuration

Reply:
done